libs/designer/src/lib/core/actions/bjsworkflow/operationdeserializer.ts

```diff
@@ -136,7 +136,9 @@ export const initializeOperationDetailsForManifest = async (
   nodeId: string,
   _operation: LogicAppsV2.ActionDefinition | LogicAppsV2.TriggerDefinition,
   isTrigger: boolean,
-  dispatch: Dispatch
+  dispatch: Dispatch,
```

Comment:
I'm a little concerned about passing `dispatch` around like this. It's generally intended to be used within the scope of a component so that other functions which might not have access to the app dispatch can still use it.

Would like input from LAUX team on how to best handle this. I think a callback argument might be the best option.

Reply:
This `dispatch` was already present in this method. I did not add it myself.